src/xtgeo/__init__.py

```diff
@@ -169,3 +170,6 @@ def _xprint(msg):
 warnings.filterwarnings("default", category=DeprecationWarning, module="xtgeo")
 
 _xprint("XTGEO __init__ done")
+
+# Remove symbols imported for internal use
+del os, sys, timeit, warnings, TIME0, DEBUG, ROXAR, _timer, _xprint
```

Comment:
Prevents someone from being able to `import xtgeo.os`

Reply:
Oh, was not aware of that 👍